### PR TITLE
Feat:"Added HomePage"

### LINKED ISSUE #3: Feat: HomePage

### PR DESCRIPTION
**Pull request by sadathshariff**
Added HomePage 
Which Includes 
1. Banner 
2. Categories that are fetched from DB
3>Footer

### CODE REVIEW

**Review by ttahm3d — COMMENTED** [review]
Looks good. The code strucutre and you import and exports are pretty neat. 

Good job.

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+    } catch (error) {
+      setError(error);
+    } finally {
+      setLoading(false);
+    }
+  };
+  useEffect(() => {
+    fetchData();
+  }, []);
+  return (
```
Comment (raman1999) [inline src/pages/Home/Home.jsx:26]: can use getAxios custom hook here for get requests

**Review by raman1999 — APPROVED** [review]
clean code written and good breakdown of components

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (sadathshariff) [reply src/pages/Home/Home.jsx:26]: Thanks, Raman planning to that

**Pull request merged**